src/components/PeopleLink/PeopleLink.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PeopleLink: React.FC<Props> = ({ person, people }) => {
+  const { slugParam } = useParams();
+
+  const { name, sex, born, died, fatherName, motherName, slug } = person;
+
+  const isMother = people.find(p => p.name === motherName);
+  const isFather = people.find(p => p.name === fatherName);
```

Comment:
```suggestion
  const mother = people.find(p => p.name === motherName);
  const father = people.find(p => p.name === fatherName);
```
we usually name boolean vars starting with `is`